app.py

```diff
@@ -120,23 +120,252 @@ def get_user_identity(user_id):
     dict_json = json.loads(json_response)
     return dict_json
 
+def change_situation(user_id, situation):
+    user_id = user_id
+    situation = situation
+    param = {
+        'user_id':user_id,
+        'situation':situation
+    }
+    url = "https://wssocinf-5-web.herokuapp.com/api/change_situation?"
+    paramStr = urllib.parse.urlencode(param)
+    r = requests.get(url + paramStr)
```

Comment:
Can you change this http method to `post` ?
The API server now can receive get method for debug but only the post method can be received originally.